### PR TITLE
Adding text to ### Storing and archiving version-controlled data

### PR DESCRIPTION
**Pull request by dylangomes**
Adding text to ### Storing and archiving version-controlled data

### CODE REVIEW

File: content/03.use-cases-a.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+
+GitHub also integrates with Zenodo, a free, long-term data archiving service funded by CERN. After linking your GitHub account to Zenodo and turning on archiving, any time a release (Box 1) is made, a snapshot of the entire repository is archived in Zenodo with a versioned, citable DOI (see `### Making code citable` below for more information).
```
Comment (Aariq) [inline content/03.use-cases-a.md:8]: I think [@doi:10.2777/298228](https://data.europa.eu/doi/10.2777/298228) might be an appropriate citation for the first sentence here?

**Review by Aariq — CHANGES_REQUESTED** [review]
Put sentences on separate lines and it's good to merge.

**Comment by dylangomes** [comment]
Do I have to close this pull request before I can resubmit one for the same section?

**Comment by dylangomes** [comment]
> Put sentences on separate lines and it's good to merge.

I can't tell if I've successfully done this or not. I know I've done it in my own branch, but not sure if it has been included in this pull request or not.

**Comment by Aariq** [comment]
Yeah, the sentences now are on separate lines!

**Review by Aariq — APPROVED** [review]

File: content/03.use-cases-a.md (hunk 1 above)
Comment (katherinehebert) [reply content/03.use-cases-a.md:14]: I like this suggestion for the sentence on line 14!

**Review by katherinehebert — APPROVED** [review]

**Pull request merged**